File: 은지/LV2/행렬의 곱셈.py
def solution(arr1, arr2):
    answer = []
    for i in range(len(arr1)):
        sub_arr = []
        for k in range(len(arr2[0])):
            sub_v = 0
            for j in range(len(arr1[0])):
                sub_v += arr1[i][j] * arr2[j][k]
            sub_arr.append(sub_v)
        answer.append(sub_arr)
    return answer

File: 은지/LV2/행렬의 곱셈.js
function solution(arr1, arr2) {
  var answer = [];
  for (let i=0; i<arr1.length; i++) {
    var sub_arr = []
    for (let k=0; k<arr2[0].length; k++) {
      var sub_v = 0
      for (let j=0; j<arr1[0].length; j++) {
        sub_v += arr1[i][j] * arr2[j][k]
      }
      sub_arr.push(sub_v)
    }
    answer.push(sub_arr)
  }
  return answer;
}